packages/excalidraw/scene/selection.ts

```diff
@@ -218,10 +218,10 @@ export const getSelectedElements = (
 
 export const getTargetElements = (
   elements: ElementsMapOrArray,
-  appState: Pick<AppState, "selectedElementIds" | "editingElement">,
+  appState: Pick<AppState, "selectedElementIds" | "editingTextElement">,
 ) =>
-  appState.editingElement
-    ? [appState.editingElement]
+  appState.editingTextElement
+    ? [appState.editingTextElement]
```

Comment:
ditto as `editingElement` was previously also multi-point arrowElement during creation. I've included `appState.newElement` in here to more or less align with previous behavior.